examples/Benchmark/replication/replication_client.cc

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ The MIT License (MIT)
+
+ Copyright (c) 2017 Grigory Nikolaenko <nikolaenko.grigory@gmail.com>
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ *******************************************************************************/
+
+#include "../../../Bricks/dflags/dflags.h"
+#include "../../../Sherlock/replicator.h"
+
+#include "entry.h"
+
+DEFINE_string(url, "127.0.0.1:8383/raw_log", "Url to subscribe to.");
+DEFINE_string(db, "replicated_db.json", "Path to load the source stream data from.");
+DEFINE_uint32(total_entries, 10000, "Entries number to replicate.");
+
+template <typename STREAM, typename... ARGS>
+void Replicate(ARGS&&... args) {
+  STREAM replicated_stream(std::forward<ARGS>(args)...);
+  using RemoteStreamReplicator = current::sherlock::StreamReplicator<STREAM>;
+  current::sherlock::SubscribableRemoteStream<benchmark::replication::Entry> remote_stream(FLAGS_url);
```

Comment:
For a standlone tool, perhaps more console output would help here? Ex. `"Connecting to the stream at $URL ..." << flush`, followed by some `"\b\b\bOK\nValidating schema ..."`, and then the number of records in the remote stream?